ping/datadog_checks/ping/data/conf.yaml.example

```diff
@@ -0,0 +1,27 @@
+init_config:
+
+instances:
+
+    ## @param host - string - required
+    ## Host to ping
+    #
+  - host: <HOST_NAME>
+
+    ## @param collect_response_time - boolean - optional - default: false
+    ## Reports network.ping.response_time in ms if true
+    #
+    # collect_response_time: false
+
+    ## @param timeout - integer - optional - default: 4
+    ## Timeout in seconds
+    #
+    # timeout: 4
```

Comment:
So actually, I'm reverting what I've said earlier.
The `-W/w` option and is in milliseconds both on Windows, Linux, and Mac.
Suggested option, replace this config block by:

```yaml
    ## @param timeout - float - optional - default: 4.0
    ## Timeout in seconds
    #
    # timeout: 4.0
```